.github/workflows/ci.yml

```
@@ -0,0 +1,12 @@
+name: Opspark Tool Ci
```

IMO, since we're already on the `opspark` tool repo, the label seems superfluous
```suggestion
name: CI
```